packages/code-du-travail-modeles/src/modeles/conventions/2596_coiffure/__tests__/indemnite-licenciement/indemnite.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { IndemniteLicenciementPublicodes } from "../../../../../publicodes";
+
+const engine = new IndemniteLicenciementPublicodes(
+  modelsIndemniteLicenciement,
+  "2596"
+);
+
+describe("Indemnité conventionnel de licenciement pour la CC 2596", () => {
+  test.each`
+    seniority | salary  | expectedCompensation
+    ${6 / 12} | ${2600} | ${0}
+    ${8 / 12} | ${2600} | ${433.33}
+    ${5}      | ${2600} | ${3250}
+    ${15}     | ${2600} | ${10833.33}
```

Comment:
Ca serait bien d'avoir les deux seniority ici. On a la seniority pour valider que l'on a le droit et la seniority pour le calcul. On plafonne la seniority de droit pour valider que l'on ne l'utilise pas pour les calculs:

```suggestion
    seniorityRight | seniority | salary  | expectedCompensation
    ${6 / 12} | ${1} | ${2600} | ${0} <-- ici je mets la seniority à 1 pour valider que l'on ne l'utilise pas pour le droit
    ${8 / 12} | ${8 / 12} | ${2600} | ${433.33}
    ${8 / 12} | ${5}      | ${2600} | ${3250} -> Ici je garde la seniority de droit à 8 mois pour valider que l'on ne l'utilise pas pour le calcul.
    ${8 / 12} | ${15}     | ${2600} | ${10833.33}
```

Reply:
ah yes :+1: